apps/android/app/src/main/res/values-ru/strings.xml

```diff
@@ -450,6 +456,18 @@
     <string name="read_more_in_github">Узнайте больше из нашего GitHub репозитория.</string>
     <string name="read_more_in_github_with_link">Узнайте больше из нашего <font color="#0088ff">GitHub репозитория</font>.</string>
 
+    <!-- MakeConnection -->
```

Comment:
Where are these used? I've checked a couple of keys but couldn't find in code, also it's for a view that's being replaced